linux/tools.Dockerfile

```diff
@@ -29,6 +29,11 @@ RUN az aks install-cli \
     && chmod +x /usr/local/bin/kubectl \
     && chmod +x /usr/local/bin/kubelogin
 
+# Install azure-developer-cli (azd)
+ENV AZD_IN_CLOUDSHELL 1
+ENV AZD_SKIP_UPDATE_CHECK 1
```

Comment:
I believe the binary checks for the environment variable here.

Reply:
`AZD_IN_CLOUDSHELL` tells the `azd` tool that it's running in Cloud Shell which enabled [Cloud Shell authentication](https://learn.microsoft.com/en-us/azure/cloud-shell/msi-authorization). 

`AZD_SKIP_UPDATE_CHECK` disables the alert that a new version of `azd` has released since the user doesn't have write access to `/opt`. Every time the install script is run the tool installs the latest available version of `azd` so each release of the image will have the most up to date stable release of `azd` at creation. 

Users who do want to install a different version of `azd` over what's installed in the image can do so by installing the binary into their home folder and putting a symlink in `~/bin` which appears earlier in the path than `/usr/local/bin` (the default install location).